docker-compose.yml

```diff
@@ -1,16 +1,15 @@
 version: "3.4"
 
 services:
-  localstack:
-    image: localstack/localstack:latest
+  s3:
+    image: minio/minio:latest
+    command: server /data
     environment:
-      - SERVICES=s3
-      - PORT_WEB_UI=8081
-      - DEFAULT_REGION=eu-west-2
-      - AWS_ACCESS_KEY_ID=foo
-      - AWS_SECRET_ACCESS_KEY=bar
+      MINIO_ACCESS_KEY: longkey
+      MINIO_SECRET_KEY: verysecretkey
+      MINIO_REGION: eu-west-2
```

Comment:
maybe mandatory to specify, but why `eu-west-2` if it's fake? would keep the default of defaults for AWS SDKs, `us-east-1`, if a value is needed here